CONTRIBUTING.md

```diff
@@ -144,104 +98,56 @@ into a single one that captures everything you're trying to do -- the history
-around you, which is just fairly standard K&R with braces on every
-compound statement. One thing to watch out for is top-level sections in
-source files that are denoted by comments and are actually indented one
-level.
-
-Hoon will be a less familiar language to many contributors.  We've published
-some [style guidelines for Hoon][hoon], but above all you should try to mimic
-the style of the code around you.  With regards to the style used throughout
-the codebase: the more recently the code was written, the more standard and
-accepted its style is likely to be.
-
-## Kernel development
+Reference any related issues, links, papers, etc. here.
+```
 
-Working on either C or non-kernel Hoon should not bring any surprises, but the
-Hoon kernel (anything under [`pkg/arvo/sys/`][sys]) is bootstrapped from a
-so-called *pill*, and must be recompiled if any changes are made. This should
-happen automatically when you make changes, but if it doesn't, the command to
-manually recompile and install the new kernel is `|reset` in `dojo`.  This
-rebuilds from the `sys` directory in the `base` desk in `%clay`.
+Tests will run automatically via GitHub Actions when you open a pull request or
+push new commits to an existing pull request.
 
-Currently, `|reset` does not reload apps like `dojo` itself, which will still
-reference the old kernel. To force them to reload, make a trivial edit to their
-main source file (under the `app` directory) in `%clay`.
+Once you've collected and addressed feedback, tests are passing, and your PR has
+been approved, merge the pull request. If you properly included the "Resolves
+#N." directive in the pull request description, merging will automatically close
+the tracking issue associated with the pull request.
 
-[arvo]: https://github.com/urbit/urbit/tree/master/pkg/arvo
-[sys]: https://github.com/urbit/urbit/tree/master/pkg/arvo/sys
+### Kernel Development and Pills
 
-## The kernel and pills
+If you're making changes to the kernel (i.e., anything in `pkg/arvo/sys`) you
+need to know about pills.
 
-Urbit bootstraps itself using a binary blob called a pill (you can see it being
-fetched from `bootstrap.urbit.org` on boot).  This is the compiled version of
-the kernel (which you can find in the `sys` directory of [Arvo][arvo]), along
-with a complete copy of the Arvo source.
+Urbit bootstraps itself from a pill (you can see it being fetched from
+`bootstrap.urbit.org` on boot).  This is the compiled version of the kernel
+(which you can find in the `sys` directory of [Arvo][arvo]), along with a
+complete copy of the Arvo source.
 
-The procedure for creating a pill is often called "soliding." It is somewhat
-similar to `|reset`, but instead of replacing your running kernel, it writes
-the compiled kernel to a file. The command to solid is:
+The procedure for creating a pill is often called "soliding." It writes the
+compiled kernel to a file. The command to solid is:
```

Comment:
fwiw I have never heard anyone say this. It's usually just "make/update a pill".

Reply:
+1 this was old text, I'll remove it and use more simple/normal language